src/main/java/com/esri/samples/symbology/read_symbols_from_mobile_style_file/ReadSymbolsFromMobileStyleFileController.java

```diff
@@ -0,0 +1,353 @@
+   */
+  @FXML
+  private void updateSymbolSize() {
+    // get the slider value and convert to a string
+    String sizeValue = Integer.toString((int) sizeSlider.getValue());
+    // display the value in the label
+    sizeLabel.setText(sizeValue + " px");
+
+    // update the preview
+    buildCompositeSymbol();
+  }
+
+  /**
+   * Adds graphics to the map view on mouse clicks.
+   * @param e mouse button click event
+   */
+  @FXML
+  private void handleMouseClicked(MouseEvent e){
+    if (e.getButton() == MouseButton.PRIMARY && e.isStillSincePress()) {
+      // convert clicked point to a map point
+      Point mapPoint = mapView.screenToLocation(new Point2D(e.getX(), e.getY()));
+
+      // create a new graphic with the point and symbol
+      Graphic graphic = new Graphic(mapPoint, faceSymbol);
+      graphicsOverlay.getGraphics().add(graphic);
+    }
+  }
+
+  /**
+   * Clears all the graphics from the graphics overlay.
+   */
+  @FXML
+  private void resetView() {
+    graphicsOverlay.getGraphics().clear();
+  }
+
+  /**
+   * Stops and releases all resources used in application.
+   */
+  public void terminate() {
+
+    if (mapView != null) {
+      mapView.dispose();
+    }
+  }
+
+  /**
+   * Shows the available symbol of the SymbolStyleSearchResult in the symbol selection list view.
+   */
+  class SymbolLayerInfoListCell extends ListCell<SymbolStyleSearchResult> {
+
+    private SymbolLayerInfoListCell() {
+      // set the cell to display only a graphic
+      setContentDisplay(ContentDisplay.GRAPHIC_ONLY);
+    }
+
+    protected void updateItem(SymbolStyleSearchResult item, boolean empty) {
+      super.updateItem(item, empty);
+
+      if (item == null || empty) {
+        // if the item in the list view is an empty item, show nothing
+        setGraphic(null);
+      } else {
+        // get the symbol from the list view entry, and create an image from it
+        ListenableFuture<Image> symbolImageFuture = item.getSymbol().createSwatchAsync(0x00000000, 1);
+        try {
+          // get the resulting image
+          Image symbolImage = symbolImageFuture.get();
+          // create and image view and display it in the cell
+          ImageView symbolImageView = new ImageView(symbolImage);
+          setGraphic(symbolImageView);
+        } catch (InterruptedException | ExecutionException e) {
+          new Alert(Alert.AlertType.ERROR, "Error creating preview image for symbol in mobile style file" + e.getMessage()).show();
+        }
+      }
+    }
+  }
+
+  /**
+   * Shows the colors in the color selection list view.
+   */
+  class ColorListCell extends ListCell<Integer> {
+    private final Rectangle rectangle;
+
+    private ColorListCell() {
+      // set the cell to display only a graphic
+      setContentDisplay(ContentDisplay.GRAPHIC_ONLY);
+      // create a rectangle to display in the cell
+      rectangle = new Rectangle(10, 10);
+    }
+
+    protected void updateItem(Integer item, boolean empty) {
```

Comment:
` @Override` ?

Reply:
Other uses of this pattern in our samples don't do an @Override here. What are the implication of leaving it out?

Reply:
I'm used to seeing compiler warnings if it's missing, but that will depend on how the IDE is set up re inspections/warnings.

More importantly, it allows the compiler to check if you're really overriding as you intended. There's a description of how that helps in the first answer in https://stackoverflow.com/questions/4736212/the-meaning-of-override-in-android-studio

Reply:
It is in Effective Java - Item 36 "Consistently use Override annotation". I don't think it meant be consistent about *not* using it!

Reply:
Thanks both, no errors popped up on my side, but I see how it's good practice to @Override. 
Will probably need to go through our samples to see if it's missing anywhere else.